whois-api/src/main/java/net/ripe/db/whois/api/rdap/domain/RdapObject.java

```diff
@@ -155,4 +173,22 @@ public String getObjectClassName() {
     public void setObjectClassName(final String value) {
         this.objectClassName = value;
     }
+
+    public void setAutnums(final List<Autnum> autnums){
+        this.autnums = autnums;
+    }
+
+    public void setIpv4(final List<Ip> ipv4Networks) {
+        if (networks == null) {
+            networks = Lists.newArrayList();
```

Comment:
Why there is a different setter for ipv4 and ipv6? IP class has a ip version which should be enough to distinguish between ipv4 and ipv6. We should use the one setter method as its ultimately only one IP class